documentation/Using-the-SDK/Creating-your-plugin.md

```diff
@@ -4,33 +4,34 @@ This document outlines how to use the Performance Toolkit SDK (SDK) to create
 
-Before creating a plugin, it is recommended to read [the overview of the SDK's architecture](../Architecture/Overview.md).
+Before creating a plugin, it is recommended to read [the overview of the SDK's architecture](../Architecture/README.md).
 
 Creating a plugin can be outlined into 4 distinct steps:
-1. [Creating the Project](#creating-the-project)
+- [Creating an SDK Plugin](#creating-an-sdk-plugin)
+  - [Creating the Project](#creating-the-project)
     - [Requirements](#requirements)
     - [Creating Your Project](#creating-your-project)
     - [Configuring Your Project](#configuring-your-project)
-        * [Adding the Microsoft.Performance.SDK NuGet Package](#adding-the-microsoftperformancesdk-nuget-package)
-        * [Picking your SDK version](#picking-your-sdk-version)
-        * [Installing WPA for Debugging](#installing-wpa-for-debugging)
-        * [Setup for Debugging Using WPA](#setup-for-debugging-using-wpa)
-2. [Creating a ProcessingSource](#creating-a-processingsource)
+      - [Adding the Microsoft.Performance.SDK NuGet Package](#adding-the-microsoftperformancesdk-nuget-package)
+      - [Picking your SDK version](#picking-your-sdk-version)
+      - [Installing WPA for Debugging](#installing-wpa-for-debugging)
+      - [Setup for Debugging Using WPA](#setup-for-debugging-using-wpa)
+  - [Creating a ProcessingSource](#creating-a-processingsource)
     - [Creating a ProcessingSource class](#creating-a-processingsource-class)
     - [Decorating your ProcessingSource with the ProcessingSourceAttribute](#decorating-your-processingsource-with-the-processingsourceattribute)
     - [Decorating your ProcessingSource with a DataSourceAttribute](#decorating-your-processingsource-with-a-datasourceattribute)
     - [Implementing the required ProcessingSource methods](#implementing-the-required-processingsource-methods)
-    - [(Optional) Adding About Information](#adding-about-information)
-3. [Choosing a Plugin Framework](#choosing-a-plugin-framework)
-4. [Creating a CustomDataProcessor and Tables](#creating-a-customdataprocessor-and-tables)
+    - [Adding About Information](#adding-about-information)
+  - [Choosing a Plugin Framework](#choosing-a-plugin-framework)
+  - [Creating a CustomDataProcessor and Tables](#creating-a-customdataprocessor-and-tables)
 
 ---
 
 ## Creating the Project
 
 Plugins are created as C# class libraries that get dynamically loaded by the SDK runtime. To begin your plugin creation, we will first walk through creating a C# project.
 
-> :information_source: The SDK team is actively working on creating a dotnet template to simplify creating your project and writing the neccessary plugin boilerplate code. 
+*Note*: The SDK team is [actively working](https://github.com/microsoft/microsoft-performance-toolkit-sdk/issues/190) to create a dotnet template to simplify creating your project and writing the necessary plugin boilerplate code. 
```

Comment:
Why the change from `> :information_source:`?

Reply:
It doesn't show up well in code. Is it a github specific extension to markdown?

Reply:
Yep, it looks like this:
![image](https://user-images.githubusercontent.com/68865158/152606681-cf0b3e6e-6b19-4971-8b42-86908decfec0.png)

Reply:
the `:emoji:` is GitHub specified, `>` is the general markdown for block quotes

Reply:
yeah, looks like it is github specific. do we still want to use it?

Reply:
ah, i'll use the unicode character and the >